rs/tests/message_routing/BUILD.bazel

```diff
@@ -18,6 +18,20 @@ system_test(
     deps = DEPENDENCIES + ["//rs/tests"],
 )
 
+system_test(
+    name = "memory_safety_test",
+    flaky = True,
```

Comment:
Do we need the `flaky = True` here?

Reply:
I'm not sure. I haven't seen it fail, but I suspected it's there for a reason. I was planning planning on looking at some runs once it's in hourly and reassess. But if you prefer I can also set it to false and flip it if things start failing.

Reply:
I think at some point `flaky = True` was the default in the testing framework. When the testing framework itself became less flaky, the default was changed someone went through all the tests and manually added `flaky = True` for all the tests so that teams can individually decide when to mark a test non flaky. So I think it may just be a leftover. Lets go for `flaky = False` and see if things become a pain.

Sidenote: we should probably also take an action item to remove the `flaky = True` for the other tests in this file where possible. [not in this PR]

Reply:
Set to False.